flake8_author.py

```diff
@@ -44,7 +44,10 @@ def add_options(cls, parser):
             **extra_kwargs)
 
     @classmethod
-    def parse_options(cls, options):
+    def parse_options(cls, options, reset=True):
+        if reset:
+            cls.options = {}
```

Comment:
`reset` doesn't appear to be part of the official flake8 extension interface, and it isn't used elsewhere in this change, unless I missed something?

Reply:
I had to add this, because when running the tests it was holding over the pattern from the previous run. I'll figure out another way to do this.

Reply:
We can add a `tearDown` method to the test class that resets the options between test runs:

```py
def tearDown(self):
    Checker.options = {}
```